Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Buttons/ButtonIconSet.cs

```diff
@@ -0,0 +1,705 @@
+﻿using TMPro;
+using UnityEngine;
+using System.Text;
+using System.Collections.Generic;
+using UnityEngine.TextCore;
+using System;
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEditor.Sprites;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    [CreateAssetMenu(fileName = "IconSet", menuName = "Mixed Reality Toolkit/IconSet")]
+    public class ButtonIconSet : ScriptableObject
```

Comment:
This `public` class and some of its `public` properties and methods are missing summary tags